lib.nix

```diff
@@ -77,4 +77,13 @@ in lib // (rec {
     philippKey
     vasilisKey
   ]);
+
+  buildSlaveKeys = with ssh-keys; {
+    macos = devOpsKeys ++ [
+      hydraBuildFarmKey
+      rodneyRemoteBuildKey
```

Comment:
`rodneyRemoteBuildKey` for testing?

Reply:
This is because in my config I want to use a separate ssh key for nix remote builders. Is this OK?